### PR TITLE
Enough to deposit files

### PR DESCRIPTION
**Pull request by rlamacraft**
I've implemented just enough of the Zenodo API to be able to deposit a file. I think this should be enough to implement our repository interface, but I'm not certain without beginning that work, so I thought this would be a good checkpoint to make sure there aren't any glaring issues.

### CODE REVIEW

File: src/main/java/com/researchspace/zenodo/model/ZenodoDeposition.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+/**
+ * A Zenodo Deposition, as returned by the API.
+ */
+@Data
+@NoArgsConstructor
```
Comment (mKowalski256) [inline src/main/java/com/researchspace/zenodo/model/ZenodoDeposition.java]: public non-args constructor is default in java, so if class just need that, then no need for annotation

File: src/main/java/com/researchspace/zenodo/model/ZenodoDeposition.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class ZenodoDeposition {
+
+  /**
+   * Various endpoints that can be invoked to perform operations on the
+   * Deposition.
+   */
+  @Data
+  @NoArgsConstructor
```
Comment (mKowalski256) [inline src/main/java/com/researchspace/zenodo/model/ZenodoDeposition.java]: same here

**Review by mKowalski256 — APPROVED** [review]
Looks good to me.

**Comment by rlamacraft** [comment]
@mKowalski256 Unless you have anything else to comment on, I'm going to look at merging this

**Comment by mKowalski256** [comment]
Yes, looks good.

**Pull request merged**